packages/libs/react-ui/src/components/Form/Select/SelectButton.tsx

```diff
@@ -38,6 +43,7 @@ function SelectButtonBase<T extends object>(
   return (
     <button
       {...mergeProps(buttonProps, dataProps, hoverProps)}
+      id="select-button"
```

Comment:
Should this be an optional prop where this `id` is the default/fallback? If you have multiple select components on a page it would be semantically incorrect 😁

Reply:
Yhea I use this to target a selector but indeed, what if you have multiple.. Rookie mistake.
I'll change this to a data attribute.